netlify.toml

```diff
@@ -2,3 +2,9 @@
   base = "site/"
   command = "jekyll build"
   publish = "site/_site"
+
+# Using the * -> :splat rule should redirect anything under master to the matching file under main
+[[redirects]]
+  from = "docs/master/*"
```

Comment:
@nrb i _think_ you might just need to have a / at the beginning of the to/from values. hoping its that straight forward :)

Reply:
Thanks, that looks like it worked!